src/context-selection/attribute-option-combo-selector-bar-item/use-category-combination.js

```diff
@@ -27,6 +27,6 @@ export default function useCategoryCombination() {
         called: !isIdle,
         loading,
         error,
-        data: data?.dataSet.categoryCombo,
+        data: data?.dataSet?.categoryCombo,
```

Comment:
How come this change is required, this should always be in the response?

Reply:
Yeah I was just sketching. I'd added guards everywhere because I didn't want to take the time to figure out which hooks were all right and which weren't.